.github/ISSUE_TEMPLATE/bug-report.yaml

```
@@ -0,0 +1,44 @@
+name: Bug report
+description: The mod doesn't work? Use this form to open a new bug report
+title: "[Bug]: <title>"
+type: bug
+body:
+  - type: markdown
+    attributes:
+      value: Please fill out this form to help us solve your issue
+
+  - type: input
+    id: mas-version
+    attributes:
+      label: Mod version
+      description: What version of MAS are you using? You can find mod version in the settings or in the logs
+      placeholder: 1.0.0
+    validations:
+      required: true
+
```

```suggestion
  - type: dropdown
    id: mas-unstable
    attributes:
      label: Unstable
      description: Are you running unstable version of MAS?
      options:
        - "Yes"
        - "No"
        - Unsure
```

We could add a flag/dropdown to notify if using unstable (if that matters)